forui/lib/src/widgets/bottom_navigation_bar/bottom_navigation_bar.dart

```diff
@@ -1,82 +1,76 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
-import 'package:collection/collection.dart';
 import 'package:meta/meta.dart';
 
 import 'package:forui/forui.dart';
 import 'package:forui/src/foundation/tappable.dart';
 
-part 'bottom_navigation_bar_item.dart';
-
 /// A bottom navigation bar.
 ///
 /// A bottom navigation bar is usually present at the bottom of root pages. It is used to navigate between a small
 /// number of views, typically between three and five.
 ///
 /// See:
 /// * https://forui.dev/docs/bottom-navigation-bar for working examples.
-/// * [FBottomNavigationBarStyle] for customizing a card's appearance.
+/// * [FBottomNavigationBarStyle] for customizing a bottom navigation bar's appearance.
+/// * [FBottomNavigationBarItem] for the items in a bottom navigation bar.
 class FBottomNavigationBar extends StatelessWidget {
   /// The style.
   final FBottomNavigationBarStyle? style;
 
-  /// The items.
-  final List<Widget> items;
-
   /// A callback for when an item is selected.
   final ValueChanged<int>? onChange;
 
+  /// The index.
+  ///
+  /// ## Contract
+  /// Throws [AssertionError] if [index] is not null and is negative.
+  final int? index;
```

Comment:
Why not make int non-nullable?

Reply:
Good catch, I could probably use negative index to represent no selected